settings.py

```diff
@@ -18,12 +18,13 @@
 if LOGGING_TO_TELEGRAM:
     from telegram_handler import TelegramHandler
 
-LOGGING_TO_FILE = False
+LOGGING_TO_FILE = True
```

Comment:
Leave it false. If you set it True while debugging you can set it back before committing